native/packages/react-native-bpk-component-button-link/src/BpkButtonLink.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
+import { View, Platform } from 'react-native';
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+
+import { withTheme } from 'react-native-bpk-theming';
+import BpkIcon from 'react-native-bpk-component-icon';
+import BpkText from 'react-native-bpk-component-text';
+
+import {
+  themeAttributesSupplied,
+  getStyleForElement,
+  textStyle,
+  iconStyle,
+} from './utils';
+
+import {
+  type CommonProps,
+  type IconAlignment,
+  COMMON_PROP_TYPES,
+  COMMON_DEFAULT_PROPS,
+  ICON_ALIGNMENTS,
+} from './common-types';
+
+import BpkButtonLinkContainer from './layout/BpkButtonLinkContainer';
+
+type Props = {
+  ...$Exact<CommonProps>,
+  iconAlignment: IconAlignment,
+  large: boolean,
+};
+
+const BpkButtonLink = (props: Props) => {
```

Comment:
Shouldn't this just be a prop for button? What's the reasoning for it being a separate component. In my head if anything this variant is just the secondary button style

Reply:
Just because it's easier. Look at the nic of our existing button component as well as the web variant. A little bit of duplication goes a long way.